Modules/_testcapimodule.c

```diff
@@ -1205,7 +1205,8 @@ test_k_code(PyObject *self)
     value = PyLong_AsUnsignedLongMask(num);
     if (value != (unsigned long)-0x42)
         return raiseTestError("test_k_code",
-            "PyLong_AsUnsignedLongMask() returned wrong value for long 0xFFF...FFF");
+                              "PyLong_AsUnsignedLongMask() returned wrong "
+                              "value for long -0xFFF..000042");
```

Comment:
`0xFFF...000042` looks strange. Is this really the right value? (And what does the `...` mean here?)